Source/Common/Config.cpp

```diff
@@ -181,6 +182,7 @@ AppLoader::AppLoader(const fextl::string& Filename, FEXCore::Config::LayerType T
 }
 
 void AppLoader::Load() {
+  LogMan::Msg::DFmt("[AppLoader] Loading config file: {}", Config.c_str());
```

Comment:
fmt can format strings, so there's no need for `.c_str()`:
```suggestion
  LogMan::Msg::DFmt("[AppLoader] Loading config file: {}", Config);
```